skrub/_expressions/_estimator.py

```diff
@@ -477,11 +536,24 @@ def cross_validate(pipeline, environment, **kwargs):
 
     Parameters
     ----------
+    expr : skrub expression
+        The expression to cross-validate, the expression from which the default
+        pipeline and data are created. If both ``pipeline`` and ``environment``
+        are provided, ``expr`` is unused and should not be provided.
+
     pipeline : skrub pipeline
-        A pipeline generated from a skrub expression.
+        A pipeline generated from a skrub expression. By default, it is
+        obtained from ``expr`` by calling :meth:`Expr.skb.get_pipeline()`. Note
+        that this creates a pipeline that uses the default value for each of
+        the (hyperparameter) choices. To perform hyperparameter search, replace
+        this default value with a pipeline created by calling
+        :meth:`Expr.skb.get_randomized_search` or
+        :meth:`Expr.skb.get_grid_search`.
 
     environment : dict
-        Bindings for variables contained in the expression.
+        Bindings for variables contained in the expression. By default, it is
```

Comment:
Maybe replace "bindings" with "dictionary"? 

```suggestion
        Dictionary containing the variables used in the expression. By default, it is
```